src/index.tsx

```diff
@@ -1110,13 +1110,12 @@ class App extends React.Component<{}, AppState> {
           id="canvas"
           width={window.innerWidth - CANVAS_WINDOW_OFFSET_LEFT}
           height={window.innerHeight - CANVAS_WINDOW_OFFSET_TOP}
-          onWheel={e => {
-            e.preventDefault();
-            const { deltaX, deltaY } = e;
-            this.setState(state => ({
-              scrollX: state.scrollX - deltaX,
-              scrollY: state.scrollY - deltaY
-            }));
+          ref={canvas => {
+            if (canvas) {
+              canvas.addEventListener("wheel", this.handleWheel, {
```

Comment:
You never clean it up. Can you do it on componentDidMount and componentWillUnmount instead?